src/bin/home/main.rs

```diff
@@ -0,0 +1,49 @@
+use clap::Parser;
+use colored::Colorize;
+use home;
+use std::env;
+use std::path::Path;
+use std::process;
+
+#[derive(Parser, Debug)]
+#[command(
+    bin_name = "sdk home",
+    about = "sdk subcommand to output the path of a specific candidate version"
+)]
+struct Args {
+    #[arg(short, long)]
+    candidate: String,
+
+    #[arg(short, long)]
+    version: String,
+}
+
+fn main() {
+    let args = Args::parse();
+    let candidate = args.candidate;
+    let version = args.version;
+    let sdkman_dir = match env::var("SDKMAN_DIR") {
+        Ok(dir) => dir,
+        Err(_) => home::home_dir().unwrap().to_str().unwrap().to_string(),
+    };
+
+    let candidates_file = format!("{}/var/candidates", sdkman_dir);
+    let valid_candidates = std::fs::read_to_string(candidates_file.as_str()).expect("msg");
```

Comment:
More descriptive error message maybe?

Reply:
Good catch. I've also added a test for this scenario.